src/main/java/org/cloudfoundry/community/servicebroker/brooklyn/service/BrooklynServiceInstanceBindingService.java

```diff
@@ -87,7 +87,7 @@ public CreateServiceInstanceBindingResponse createServiceInstanceBinding(CreateS
         
 		String childEntityId = null;
 		if (ServiceUtil.getFutureValueLoggingError(admin.hasEffector(entityId, entityId, "bind"))) {
-			Future<String> effector = admin.invokeEffector(entityId, entityId, "bind", "never", ImmutableMap.of());
+			Future<String> effector = admin.invokeEffector(entityId, entityId, "bind", "never", request.getParameters() != null ? request.getParameters() : ImmutableMap.of());
```

Comment:
What happens in the parameters provided are not valid for the bind effector? e.g if bind doesn't support params but some are provided, or if the keys have a typo?